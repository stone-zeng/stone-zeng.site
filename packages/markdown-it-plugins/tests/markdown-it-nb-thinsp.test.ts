import { test, expect } from 'vitest'
import MarkdownIt from 'markdown-it'

import MarkdownItNbThinsp from '../src/nb-thinsp'

const md = new MarkdownIt()
md.use(MarkdownItNbThinsp)

test('No-break thin space', () => {
  expect(md.renderInline('1\\,GB')).toMatchSnapshot()
  expect(md.renderInline('\\; is semicolon')).toMatchSnapshot()
  expect(md.renderInline('`\\,` is \\\\,')).toMatchSnapshot()
})
