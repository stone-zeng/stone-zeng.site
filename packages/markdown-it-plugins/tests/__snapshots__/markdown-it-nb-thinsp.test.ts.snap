// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`No-break thin space 1`] = `"1&#x202F;GB"`;

exports[`No-break thin space 2`] = `"; is semicolon"`;

exports[`No-break thin space 3`] = `"<code>\\,</code> is \\,"`;
